generators/app/index.js

```diff
@@ -1,123 +1,125 @@
 
-		this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('img/.gitkeep'));
-		this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/directives/.gitkeep'));
-		this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/factories/.gitkeep'));
-		this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/filters/.gitkeep'));
-		this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/views/.gitkeep'));
-		this.fs.copy(getTemplate('js/helpers.js'), this.destinationPath('src/js/helpers.js'));
-		this.fs.copy(getTemplate('js/modernizr.js'), this.destinationPath('src/js/modernizr.js'));
-		this.fs.copy(getTemplate('view/view.html'), this.destinationPath('src/views/Home/Home.html'));
-		this.fs.copy(getTemplate('view/view.js'), this.destinationPath('src/views/Home/Home.js'));
-		this.fs.copy(getTemplate('view/view.scss'), this.destinationPath('src/views/Home/Home.scss'));
+    mkdirp.sync(this.destinationPath('img'));
+    mkdirp.sync(this.destinationPath('src/scss/'));
+    mkdirp.sync(this.destinationPath('src/directives'));
+    mkdirp.sync(this.destinationPath('src/factories'));
+    mkdirp.sync(this.destinationPath('src/filters'));
+    mkdirp.sync(this.destinationPath('src/views'));
+    mkdirp.sync(this.destinationPath('src/views/Home'));
+    mkdirp.sync(this.destinationPath('src/js'));
 
-		this.directory(getTemplate('scss'), this.destinationPath('src/scss/'));
+    this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('img/.gitkeep'));
+    this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/directives/.gitkeep'));
+    this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/factories/.gitkeep'));
+    this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/filters/.gitkeep'));
+    this.fs.copy(getTemplate('.gitkeep'), this.destinationPath('src/views/.gitkeep'));
+    this.fs.copy(getTemplate('js/helpers.js'), this.destinationPath('src/js/helpers.js'));
+    this.fs.copy(getTemplate('js/modernizr.js'), this.destinationPath('src/js/modernizr.js'));
+    this.fs.copy(getTemplate('view/view.html'), this.destinationPath('src/views/Home/Home.html'));
+    this.fs.copy(getTemplate('view/view.js'), this.destinationPath('src/views/Home/Home.js'));
+    this.fs.copy(getTemplate('view/view.scss'), this.destinationPath('src/views/Home/Home.scss'));
 
-		this.fs.copy(getTemplate('.htaccess'), this.destinationPath('.htaccess'));
-		this.fs.copy(getTemplate('editorconfig.txt'), this.destinationPath('.editorconfig'));
-		this.fs.copy(getTemplate('data.json'), this.destinationPath('data.json'));
-		this.fs.copy(getTemplate('gulpfile.js'), this.destinationPath('gulpfile.js'));
-		this.fs.copy(getTemplate('readme.md'), this.destinationPath('readme.md'));
-		this.fs.copy(getTemplate('js/data-factory.js'), this.destinationPath('src/factories/'+properties.prefixedName+'data/'+properties.prefixedName+'data.factory.js'));
+    copydir.sync(getTemplate('scss'), this.destinationPath('src/scss/'));
```

Comment:
`this.directory()` was removed in generator 1.0.0. `copy-dir` package handles the copy now.